packages/passport/sdk/src/errors/handleTrackError.ts

```diff
@@ -0,0 +1,10 @@
+import { trackError } from '@imtbl/metrics';
+
+const moduleName = 'passport';
+
+export const handleError = (method: string, error: unknown) => {
```

Comment:
nit: given this is used as `throw handleError(...)` it might read better if this was named wrappedError so it reads as `throwWrappedError` or similar